pkg/kubelet/kuberuntime/kuberuntime_container.go

```diff
@@ -404,6 +404,20 @@ func (m *kubeGenericRuntimeManager) updateContainerResources(pod *v1.Pod, contai
 	return err
 }
 
+func (m *kubeGenericRuntimeManager) updatePodSandboxResources(sandboxID string, pod *v1.Pod) error {
+	podResourcesRequest := m.generatePodSandboxResources(sandboxID, pod)
+	if podResourcesRequest == nil {
+		return fmt.Errorf("sandboxID %q updatePodSandboxResources failed: cannot generate resources config", sandboxID)
+	}
+
+	ctx := context.Background()
+	_, err := m.runtimeService.UpdatePodSandboxResources(ctx, podResourcesRequest)
```

Comment:
What happens if the remote service hasn't implemented this method? I assume this just returns an error. Is there any way to destinguish a "not impemented" error here?